pandas/tseries/holiday.py

```diff
@@ -345,8 +345,8 @@ class AbstractHolidayCalendar(metaclass=HolidayCalendarMetaClass):
     """
 
     rules = []  # type: List[Holiday]
-    start_date = Timestamp(datetime(1970, 1, 1))
-    end_date = Timestamp(datetime(2030, 12, 31))
+    start_date = Timestamp(datetime(1960, 1, 1))
+    end_date = Timestamp(datetime(2080, 12, 31))
```

Comment:
Why not wider? 🤷‍♂

Reply:
> I have no problem with wider. I was just concerned there might be some performance hit with a wider date range. My usage is primarily calculating payment dates for bonds and swaps, where going out to 50 or 60 years will be fine in general, but in principle it would certainly be better to go back to, say, 1900 and out to, say, 2200. We have certainly seen bonds issued with 100 years to maturity.

Let's go with 1900 - 2200

Reply:
> I’m fine with just extending the end date for now.

That's fine.  Let's go with 1970 - 2200 then.

Reply:
Do you need me to change my code and do a new pull request?

Reply:
Even better: just push new changes to your branch.  It will update this PR.